frontend/src/app/about/page.tsx

```
@@ -1,37 +1,73 @@
             <p className="text-[28px] leading-[36px] md:text-[26px] md:leading-[34px] lg:text-[28px] lg:leading-[36px] font-normal text-center text-white w-full">
-              Descubre la historia, la pasión y las personas detrás de los productos que transformarán tu cabello.
+              {description}
             </p>
           </div>
 
           <div className="flex flex-row items-center gap-4">
-            <button className="flex justify-center items-center h-12 hover:opacity-90 transition-opacity">
+            <button 
+              onClick={handleViewProducts}
+              className="flex justify-center items-center h-12 hover:opacity-90 transition-opacity"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)